docs/yymnist/README.md

```diff
@@ -0,0 +1,80 @@
+README
+==========
+
+# Introduction
+
+This document is intended to walk a user through the necessary steps to run object detection on the [yymnsit](https://github.com/cmu-sei/yymnist) dataset with Juneberry.
+
+# Setup 
+Install opencv, Juneberry, and pull down the yymnsit repo
+<br>
+`git config --global http.proxy http://cloudproxy.sei.cmu.edu:80/`
+`
+git clone https://github.com/cmu-sei/yymnist.git
+`
+# Create your datasets
+The yymnist repository has a script that allows a user to create reproducible datasets to test.
+
+1. We first need to generate our train/test/val datasets. 
+
+For this example we will do a 80/10/10 split respectively.
```

Comment:
Work with Anusha to figure out what the numbers really should be. We probably want to change the dataset names to be more generikc.  For example, the annotation file is called "coco_train1k.json" so we should probably rename to "coco_train.json" or "annotations_train.json" or something. I don't think she paid particular attention to naming.

Reply:
Still working with Anusha,
Other things mentioned in this comment have been addressed here: 757dabf

Reply:
Got insight from Anusha and have settled on using a 1000 / 250 / 250 split based on her experience.

Reply:
Got insight from Anusha and have settled on using a 1000 / 250 / 250 split based on her experience.